README.md

```diff
@@ -47,7 +47,7 @@ Galaxy image.
    host.
 
     ```
-    docker run --rm -e POSTGRES_DB=galaxy -e POSTGRES_USER=galaxydbuser \
+    docker run --rm -d -e POSTGRES_DB=galaxy -e POSTGRES_USER=galaxydbuser \
```

Comment:
Daemonizing it is ok but personal preference was to be able to easily stop the container after the build is complete from an active shell...